.github/labels.yml

```diff
@@ -1,18 +1,20 @@
+disabled_actions:
+  - triage
+  - review
+  - wip
+  - lgtm
```

Comment:
What does `disabled_actions` do? I couldn't find any documentation on it.

Reply:
https://github.com/dimagi/label-bot#disabling-actions

Reply:
This just prevents PR's getting status updates for workflows that we aren't using.